plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1891,7 +1884,7 @@ public void testCreateOrReplaceTableTableCommentIsRemoved()
     @Test
     public void testCreateOrReplaceTableWithEnablingCdcProperty()
     {
-        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_create_or_replace_with_cdv", " (a BIGINT)")) {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_create_or_replace_with_cdccreateTestingFileHiveMetastore", " (a BIGINT)")) {
```

Comment:
Is the name `test_create_or_replace_with_cdccreateTestingFileHiveMetastore` intentional?

Just thinking that maybe you intended to use `test_create_or_replace_with_cdc`

Reply:
It was a mis-copy